lang/py/src/avro/lib/timezones.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+from datetime import timedelta
+from datetime import tzinfo
+
+
+class UTCTzinfo(tzinfo):
+  def utcoffset(self, dt):
+    return timedelta(0)
+
+  def tzname(self, dt):
+    return "UTC"
+
+  def dst(self, dt):
+    return timedelta(0)
+
+utc = UTCTzinfo()
+
+
+# Test Time Zone with fixed offset and no DST
+class TSTTzinfo(tzinfo):
```

Comment:
Oh my bad , my phone's auto correct messed up the test. It was "yes we.... "

On Jul 29, 2016 8:07 PM, "Chia-chi Lin" <notifications@github.com> wrote:

> In lang/py/src/avro/lib/timezones.py
> <https://github.com/Yelp/avro/pull/17#discussion_r72880894>:
>
> > +
> > +class UTCTzinfo(tzinfo):
> > +  def utcoffset(self, dt):
> > +    return timedelta(0)
> > +
> > +  def tzname(self, dt):
> > +    return "UTC"
> > +
> > +  def dst(self, dt):
> > +    return timedelta(0)
> > +
> > +utc = UTCTzinfo()
> > +
> > +
> > +# Test Time Zone with fixed offset and no DST
> > +class TSTTzinfo(tzinfo):
>
> Ok, thanks for the context. :) I'm not familiar Yaeswe. What is it btw?
> Also, it might be better to separate the code for actual library and the
> code for tests in the future. It's a little confusing to mix them together.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Yelp/avro/pull/17/files/14266470952a8af9f7aa8c0f678ef900be791714#r72880894>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE4IV1ArBzsbxvW3d71hVNb2R5Wjlkgyks5qar_-gaJpZM4JXxAM>
> .
>